samples/03_org_administrators/clone_storymap_version2.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p><code>f'{story_map._gis._portal.resturl}content/users/{item._user_id}/items/{item.itemid}/resources/export'</code></p><p> </p><p>The <code>story_map</code> should be replaced with <code>item</code> in this function. The current code will work, but it is not a recommended pattern to seek out of the function in to global variables and use it while you have the same object passed as a parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Got it. Made the change.</p>